css-typed-om/Overview.bs

```diff
@@ -2847,10 +2847,49 @@ of serializations of the contained {{CSSTransformComponent}} objects.
 
 {{CSSFontFaceValue}} Serialization {#fontfacevalue-serialization}
 -----------------------------------------------------------------
-
+ 
 {{CSSFontFaceValue}} objects are serialized to the value of their contained
 {{CSSFontFaceValue/fontFamilyName}}.
 
+Units and Values Serialization {#unitsandvalues-serialization}
+--------------------------------------------------------------
+
+This section covers how to serialize the various unit types in CSSOM
+
+<div algorithm="serialize a 'length' value">
+
+To <dfn>serialize a 'length' value</dfn>, run these steps:
+    1. Let |values| initially be the empty [=list=].
+    2. If the <length> value has a '-' sign character append it to |values|.
+    3. Append the result of [=serialize a <length> value=] to |values|. 
+    4. If the number is '0' skip this step. Else, append the <length> 
+        value's unit identifier (a two-letter abbreviation) to |values|.
+    5. Return the result of concatenating all the items in |values|,
+        separated by " " (U+0020 SPACE).
+</div>
+
+<div algorithm="serialize a 'number' value">
+
+To <dfn>serialize a 'number' value</dfn>, run these steps:
+    1. Return the number respresented as a string.
```

Comment:
If we're trying to replace CSSOM for this, we need to go into the same level of detail as it does, particularly regarding the minimum required amount of decimal precision, and if we ever output scinot.